### ISSUE TITLE
Login Box Text 

### ISSUE THREAD

**Issue by mlefebvre5** [bywatersolutions/koha-plugin-opac-theme-galadriel#10]
The Text Color of the plugin for the Login section seems to only control "Don't have an account?" and not "Login" or "Password"

Please add the ability for the Text Color to control all the text in the Login area.

**Comment by ricofreak** [comment]
there are two options here. one for "text", controlling "login"/"password"/"dont have an account".

There is also "top text", which changes "Log into your account". 

Do we want to condense this all into a single option?

**Comment by mlefebvre5** [comment]
I don't think we want to condense it all into one option. Looking closer at the text for Login: and Password: it might actually have been changed by the "text" option but it looks different maybe because of the font size? "Don't have an account" and "Forgot your password" are larger fonts then "Login" and "Password" so maybe it's not the color that is throwing me off but the font size. See model.bywatersolutions.com for example.

The font size for "Don't have an account" is medium which is what we picked with the font setting but that might not have included Login and Password.
